### PR TITLE
added add counter modal and button

### PR DESCRIPTION
**Pull request by offeka**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,7 +10,9 @@
   "dependencies": {
     "@popperjs/core": "^2.5.4",
     "@types/bootstrap": "^5.0.1",
+    "@types/jquery": "^3.5.5",
```
Comment (Kovler10) [inline package.json:13]: uninstall pls

File: src/components/EditableCounter.vue [hunk 2]
```diff
@@ -1,135 +1,106 @@
...
     },
     ...mapState(["mode"]),
   },
   watch: {
-    maxCountLocal(newValue: number) {
+    maxCount(newValue: string, oldValue?: string) {
       if (isNaN(Number(newValue))) {
-        this.isValidNumber = false;
-        this.displayAlert = true;
+        if (!oldValue || (newValue.length > oldValue.length))
```
Comment (Kovler10) [inline src/components/EditableCounter.vue:77]: can you change the value of the input to 0 or something after displaying the message?

File: src/components/Modal.vue [hunk 3]
```diff
@@ -1,28 +1,58 @@
...
+import {Modal} from "bootstrap";
+
+interface ModalData {
+  modal?: Modal;
+}
 
 export default (Vue as VueConstructor).extend({
   name: "Modal",
-  props: {modalName: String}
+  props: {modalName: String, title: Boolean, show: Boolean, centered: Boolean},
```
Comment (Kovler10) [inline src/components/Modal.vue:24]: I would expect ```title``` to be a ```string``` not a ```boolean```

File: src/components/Modal.vue [hunk 4]
```diff
@@ -1,28 +1,58 @@
...
+  data(): ModalData {
+    return {
+      modal: undefined
+    }
+  },
+  mounted() {
+    this.modal = new Modal(this.$el)
```
Comment (Kovler10) [inline src/components/Modal.vue:31]: what is ```this.$el```?

Comment (offeka) [reply src/components/Modal.vue:31]: root component html element

**Pull request merged**